Birthday-2024-Project/Scripts/PieceLogic.gd

```diff
@@ -281,14 +281,17 @@ func _SetReturnPoint():
 	#print(global_position)
 
 func _input(event):
-	if event.is_action_pressed("GrabPiece"):
+	if event.is_action_pressed("GrabPiece") and totalCells > 0:
```

Comment:
Not sure what this condition is checking. Are there pieces with 0 cells or returning 0 cells?

Reply:
Condition added to prevent uninitialized library pieces from being included